docs/03-endpoints/api-admin/permissions.md

```diff
@@ -222,8 +223,53 @@ Therefore, it is not possible to create new default object access permissions
 for the ProjectAdmin and ProjectMember groups of a project. 
 However, the default permissions set for these groups can be modified; see below for more information.
 
+### Updating an existing Default Object Access Permission
+
+- `PUT: /admin/permissions/doap/<doap_permissionIri>` to change the attributes of an existing default object
+  access permission, identified by its IRI `<doap_permissionIri>`.
+
+This is an example of a request body to update an existing default object access permission:
+
+```json
+{
+  "forProperty" : "http://api.dasch.swiss/ontology/00FF/images/v2#hasTitle",
+  "forResourceClass": "http://api.dasch.swiss/ontology/0803/incunabula/v2#Book",
+  "forGroup": null, 
+  "hasPermissions": [
+    {
+      "additionalInformation": "http://www.knora.org/ontology/knora-admin#ProjectMember",
+      "name": "D",
+      "permissionCode": 7
+    }
+  ]
+}
+```
+
+All attributes of the default object access permission are optional and may be combined.
+
+!!! note
+    Only certain combinations of attributes are allowed. Only exactly one of the following combinations is allowed:
+
+      - `forGroup`  <!-- markdownlint-disable MD046 -->
+      - `forResourceClass`
+      - `forProperty`
+      - `forResourceClass` and `forProperty`
```

Comment:
Before, the unordered list was not rendered correctly by mkdocs. I served locally and played around until it worked, then I committed: https://github.com/dasch-swiss/dsp-api/pull/3425/commits/8f95db8123bac12e999b3b83828267935d2d3d45

Reply:
I see. ` <!-- markdownlint-disable MD046 -->` :/

Reply:
Thanks.